form_components/AppointmentForm.js

```diff
@@ -10,12 +10,14 @@ import {
 
 // const Realm = require('realm');
 
-function AppointmentForm({ addAppoint }){
+function AppointmentForm({ addAppoint, goBack }){
```

Comment:
We don't use goBack in this file, does it need to be here?
```suggestion
function AppointmentForm({ addAppoint }){
```

Reply:
it does not